src/rules/prefer-locator.ts

```diff
@@ -0,0 +1,69 @@
+import ESTree from 'estree'
+import { getStringValue, isPageMethod } from '../utils/ast'
+import { createRule } from '../utils/createRule'
+
+const pageMethods = new Set([
+  'click',
+  'dblclick',
+  'dispatchEvent',
+  'fill',
+  'focus',
+  'getAttribute',
+  'hover',
+  'innerHTML',
+  'innerText',
+  'inputValue',
+  'isChecked',
+  'isDisabled',
+  'isEditable',
+  'isEnabled',
+  'isHidden',
+  'isVisible',
+  'press',
+  'selectOption',
+  'setChecked',
+  'setInputFiles',
+  'tap',
+  'textContent',
+  'uncheck',
+])
+
+function isSupportedMethod(node: ESTree.CallExpression) {
+  if (node.callee.type !== 'MemberExpression') return false
+
+  const name = getStringValue(node.callee.property)
+  return pageMethods.has(name) && isPageMethod(node, name)
```

Comment:
Should also check `frame.click`, etc. as well

Reply:
Won't frame methods be picked up by
`export function isPageMethod(node: ESTree.CallExpression, name: string) {
  return (
    node.callee.type === 'MemberExpression' &&
    dig(node.callee.object, /(^(page|frame)|(Page|Frame)$)/) &&
    isPropertyAccessor(node.callee, name)
  )
}`

Reply:
Oh yeah, my mistake